gradle.properties

```diff
@@ -15,9 +15,9 @@ kotlin.stdlib.default.dependency=false
 # Platform
 platformSinceBuild=222
 # Develop platform
-platformVersion=2022.3
+platformVersion=2023.2
 platformType=PC
-usePlugins=python:222.4167.37
+usePlugins=python:232.10227.8
```

Comment:
I assumed this should be the latest version?

Reply:
now, the newest IntelliJ IEDA version is 233.15026.9